### PR TITLE
Complete Swedish translation

### PR DESCRIPTION
**Pull request by slicke**
I have merged in the missing strings and translated them

### CODE REVIEW

**Comment by jamorham** [comment]
Currently reporting the error: `Error: Found item String/toast_crash more than one time`

**Comment by slicke** [comment]
Oops sorry. I had two duplicate lines. Seems to build in Travis now, its waiting for the emulator to run 😊

**Comment by jamorham** [comment]
I having problems getting this reviewed because the auto-format change to whitespace makes it appear that every line is new making review much more difficult. I'm assuming you didn't change every single translated string.

Can you do this to resolve the issue:

1) Take a separate backup copy of your `strings-sv.xml` file.
2) Pull in the current master and overwrite the modified strings with the current master - I have applied android studio auto format to it to try to match your indentation.
3) Replace the `strings-sv.xml` file with your translated one
4) Push those changes up to the pull request and hopefully then it will show just the subset of changes you have actually made.

File: app/src/main/res/values-sv/strings-sv.xml [hunk 1]
```diff
@@ -1,477 +1,802 @@
...
+   <string name="re_enable_all_alerts">Återaktivera alla varningar</string>
+   <string name="update_channel_colon_space">"Uppdatera källa: "</string>
+   <string name="a_new_version_is_available">En ny version finns tillgänglig</string>
+   <string name="version_details">Versionsinformation</string>
+   <string name="download_now">Ladda ned nu</string>
+   <string name="automatically_check_for_updates">Sök automatiskt efter uppdateringar</string>
+   <string name="here_you_can_save_the_settings_to_the_external_storage_sdcard">Här kan du spara dina inställningar till ett externt lagringsmedium (t ex SD-kort).\n\nKänslig data som sparas externt kan möjligen läsas av obehöriga som har tillgång till mediet.\n\nRadera inställningarna efter att du har återimporterat dem, om du är besvärad av detta.</string>
+   <string name="save_all_settings_to_sdcard">Exportera samtliga inställningar till SD-kort</string>
+   <string name="load_all_settings_from_sdcard">Läs in samtliga inställningar från SD-kort</string>
+   <string name="delete_any_settings_on_sdcard">Ta bort samtliga inställningar på SD-kort</strin
```
Comment (hjortling) [inline app/src/main/res/values-sv/strings-sv.xml]: Typo, should be: 'Antal minuter innan samma larm utlöses efter snooze'

Comment (hjortling) [inline app/src/main/res/values-sv/strings-sv.xml]: Typo, should be: 'Antal minuter innan samma kalibreringsförfrågan ska upprepas'

**Comment by slicke** [comment]
Sorry for the late reply! Ive tried to fix the indents now

**Comment by slicke** [comment]
@jamorham I should add tht the file should be complete, thus a rewrite is fine rather than merging.

**Comment by tolot27** [comment]
@slicke @hjortling Thanks for your contribution. We translate xDrip using the semi-automated workflow system of [crowdin](https://crowdin.com). Please create an account at crowdin and add your suggestions or upload your file, there. Then, they will go through the translation workflow including proof reading.
Please report back here, when you are done and close this PR.

**Comment by tolot27** [comment]
I've merged this translations into Crowdin. Hence, you can close this PR.

**Pull request closed without merging**